### DISCUSSION TITLE
'not' option for .status() parameter

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#9541] [Ideas] [answered]
> Created by: Visiluna (pjones@visiluna.com) on 2015/06/16 22:04:13 +0000
> Votes at time of UserVoice import: 1

-----------------------------------

The current `.status()` parameter (craft.entries) allows you to specify multiple statuses. But it does not allow you to specify all-except-one. 

My use case: I want to display a list of all entries, live, pending, and expired, but I want to exclude disabled entries.

To do that now, I have to use `.status('live','pending','expired')`. Logic would dictate that I should be able to just use `.status('not disabled')`, or something like that. But that just doesn't appear to be an option.

**Comment by brandonkelly** [comment, marked as answer]
This has been added for the next release (ed24fa5a4a4d1d0741ae8c6e19040afb9a93b953).

Supported syntaxes are:

- `.status(['not', 'foo', 'bar'])`
- `.status('not, foo, bar')`
